internal/envoy/v3/listener.go

```diff
@@ -279,6 +280,12 @@ func (b *httpConnectionManagerBuilder) MaxRequestsPerConnection(maxRequestsPerCo
 	return b
 }
 
+// PerConnectionBufferLimitBytes the soft limit on size of the listener’s new connection read and write buffers
+func (b *httpConnectionManagerBuilder) PerConnectionBufferLimitBytes(perConnectionBufferLimitBytes *uint32) *httpConnectionManagerBuilder {
+	b.perConnectionBufferLimitBytes = perConnectionBufferLimitBytes
```

Comment:
The `Get` method also needs to be modified to generate the xds config for this setting

Reply:
Yep, unfortunately we've got a lot of this

Reply:
Had been thinking why nothing failed and suspected I am missing something. Will take a look and change.

Reply:
Changed the code to set it on the Listener